.github/workflows/dev.yml

```diff
@@ -241,7 +241,9 @@ jobs:
         if: "${{ github.event.pull_request || github.event.merge_group }}"
         run: |
           BASE="${{ github.event.pull_request.base.sha || github.event.merge_group.base.sha }}"
-          just build-sweep "${BASE}"
+          just debug_justfile="${{matrix.debug_justfile}}" rust="${{matrix.rust.version}}" \
+            build-sweep ${BASE}
+          printf "::notice::HEAD back to %s\n" "$(git log --oneline --no-decorate -n 1)"
```

Comment:
Alternatively, you could maybe just set a dummy author name and email in CI? I don't mind much though, I've always been a bit wary of using `git rebase` for the feature anyway so I'm OK reverting this.